src/azure-cli/azure/cli/command_modules/resource/tests/latest/tenant_level_template.json

```diff
@@ -8,7 +8,7 @@
     },
     "nestedSubId": {
       "type": "string",
-      "defaultValue": "0b1f6471-1bf0-4dda-aec3-cb9272f09590"
+      "defaultValue": "6898adc8-5045-473d-a1bf-7012564f43cb"
```

Comment:
Please do not change the `nestedSubId` of this test. If you don't have the permission to execute this test, I can rerecord yaml files instead of you

Reply:
I do not have the permissions to execute this test, please do.